crux-core/src/crux/api/alpha/TxResult.java

```diff
@@ -1,18 +1,33 @@
 package crux.api.alpha;
 
+import clojure.lang.Keyword;
+
 import java.util.Date;
+import java.util.Map;
+
+import static crux.api.alpha.Util.keyword;
 
 public class TxResult {
+    private static final Keyword TX_TIME = keyword("crux.tx/tx-time");
+    private static final Keyword TX_ID = keyword("crux.tx/tx-id");
+
     public final Date txTime;
     public final long txId;
+    private Map<Keyword, ?> txResultMap;
```

Comment:
nit: `final`

Reply:
Yeah - need to check a few places that probably need 'final'